test/unit/knex.js

```diff
@@ -98,10 +99,76 @@ describe('knex', () => {
     const knexWithParams = knex.withUserParams({ userParam: '451' });
 
     expect(knexWithParams.migrate.knex.userParams).to.deep.equal({
+      isProcessingDisabled: true,
+      postProcessResponse: undefined,
       userParam: '451',
+      wrapIdentifier: undefined,
     });
   });
 
+  it('copying does not result in duplicate listeners', () => {
```

Comment:
kind of duplicate with test "adding listener to copy does not affect base knex"

Reply:
They are different, actually. One tests for what happens after you copy, another one what happens after you mutate the copy.